src/main.js

```diff
@@ -105,7 +110,28 @@ function switchToViewSavedCovers(currentView) {
   savedView.classList.remove("hidden");
 }
 
-// We've provided two functions to get you started
+// MAKE MY BOOK BUTTON //
+createBookBtn.addEventListener("click", function(event){
+  event.preventDefault();
```

Comment:
can you explain why we need to prevent the default again?

Reply:
Yeah for sure! So the default of the "Make my book" button was to refresh the page. So the event.preventDefault() stops it from refreshing and instead invokes the function on line 116.

Reply:
Amazing, thanks for the explanation!!